controllers/apply/form-router-next/lib/validate-model.js

```diff
@@ -81,6 +81,38 @@ module.exports = function validateModel(formModel) {
         allowUnknown: true
     });
 
+    const arrayFieldTypes = [
+        'budget',
+        'checkbox',
+        'date',
+        'date-range',
+        'day-month'
+    ];
+
+    // Check that we haven't used a multipart field on a step with array values
+    // because we use a custom body parser (Formidable) which doesn't handle these
+    // in the same way as body-parser (eg. using qs library). Throws an error if
+    // we attempt to add an array field to one of these steps.
+    formModel.sections = formModel.sections.map(section => {
```

Comment:
This feels a bit janky but I like the reassurance of it failing the build if we forget about these constraints.